python_functionality/homework.py

```diff
@@ -25,7 +28,11 @@ def task_2_remove_dict_fields(data: DT, redundant_keys: List[str]) -> DT:
        remove_dict_field([{'name': 'Alex', 'age': 26}, {'name': 'denys', 'age': 89}], 'age')
         >>> [{'name': 'Alex'}, {'name': 'denys'}]
     """
-    pass
+    for redundant_key in redundant_keys:
+        for dic in data:
+            if dic.get(redundant_key):
+                del dic[redundant_key]
```

Comment:
It will be better if you will use list comp. You will no longer need to remove a key